src/App.js

```diff
@@ -22,27 +22,26 @@ function App() {
     setVisibleMentorIds(searchResults.map((result) => result.item.id));
   };
 
-  const [isModalActive, setIsModalActive] = useState(false);
+  const [isModalOpen, setIsModalOpen] = useState(false);
   const [activeMentorId, setActiveMentorId] = useState(null);
 
   const readMoreHandler = (mentorId) => {
-    setIsModalActive(true);
+    setIsModalOpen(true);
     setActiveMentorId(mentorId);
   };
 
   const closeHandler = () => {
-    setIsModalActive(false);
+    setIsModalOpen(false);
     setActiveMentorId(null);
   };
 
   return (
     <div className="container">
       <Header onSearch={searchHandler} />
 
-      {isModalActive && (
+      {isModalOpen && (
         <ProfileModal
-          key={activeMentorId}
-          isModalActive={isModalActive}
+          isOpen={isModalOpen}
```

Comment:
I think you haven’t updated the component file to reflect this.

Reply:
Oops, I thought I added that in lmao